Library/Homebrew/extend/ARGV.rb

```diff
@@ -125,7 +125,7 @@ def git?
   end
 
   def homebrew_developer?
-    include?("--homebrew-developer") || !ENV["HOMEBREW_DEVELOPER"].nil?
+    !ENV["HOMEBREW_DEVELOPER"].nil?
```

Comment:
We could maybe add this on `ENV` and keep this method for compatibility; then at some point in the future remove it when we’ll be used to the new API?